src/connectors/articleService.ts

```diff
@@ -1403,40 +1319,100 @@ export class ArticleService extends BaseService {
   }: {
     id: string
     order?: string
-    state?: string
-    after?: any
-    before?: any
+    after?: { type: NODE_TYPES; id: string }
+    before?: { type: NODE_TYPES; id: string }
     first?: number
     includeAfter?: boolean
     includeBefore?: boolean
     articleOnly?: boolean
   }) => {
-    const query = this.makeResponseQuery({ id, order, state, articleOnly })
+    const subQuery = this.knexRO
+      .select(
+        this.knexRO.raw('COUNT(1) OVER() AS total_count'),
+        this.knexRO.raw('MIN(created_at) OVER() AS min_cursor'),
```

Comment:
is there any issue since `created_at` might not be unique?

Reply:
and need to confirm if `created_at` is indexed.

Reply:
postgres datetime's  resolution is [1 microsecond](https://www.postgresql.org/docs/12/datatype-datetime.html) (1/1,000,000 second),  so the chance that items share exactly  same  `created_at` is extremely low

Reply:
>and need to confirm if created_at is indexed.

both indexed in `comment`  and `article` table

Reply:
> is there any issue since created_at might not be unique?
and `row_number` way (order by `created_at`)  have same issue when `created_at` not unique